.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
-.env
+/.env
```

Comment:
Why?

Reply:
because the previous attempt at naming docker compose files needed to NOT ignore a env file in a directory.
this is now outdated to it can be reverted back